### PR TITLE
initial header and footer

### PR DESCRIPTION
**Pull request by lindsayplatt**
@jiwalker-usgs review plz?

#32

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -150,3 +168,39 @@ publish:
...
+    id: footer-section
+    template: footer
+    publisher: footer
+    depends: [ "header-footer-css", "vizlab-logo" ]
+    context:
+      logo: vizlab-logo
+    blogsInFooter: FALSE
+    vizzies:
+      - name: Microplastics in the Great Lakes
+        org: USGS-VIZLAB
```
Comment (ldecicco-USGS) [inline viz.yaml:195]: maybe needs an indent here?

File: viz.yaml [hunk 2]
```diff
@@ -124,6 +123,25 @@ visualize:
...
+    id: usgs-logo
+    location: images/usgsLogo.png
+    mimetype: image/png
+    title: "USGS Logo"
+    alttext: "USGS Logo"
+  -
+    id: vizlab-favicon
+    location: images/favicon.ico
+    mimetype: image/png
+    comment: saying it is a png to get it in right spot, need handler for ico
```
Comment (jiwalker-usgs) [inline viz.yaml:141]: This is something we want to work into the vizlab package.  Maybe I'll try to work that in before release.

File: viz.yaml (hunk 1 above)
Comment (jiwalker-usgs) [reply viz.yaml:195]: This is right, the other `-` just have a newline after them

**Review by jiwalker-usgs — APPROVED** [review]
Looks good to me, there are a couple trailing issues that we should push up to vizlab (ico)

**Pull request merged**